os/arch/arm/src/armv7-m/up_busfault.c

```diff
@@ -0,0 +1,122 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+/****************************************************************************
+ * arch/arm/src/armv7-m/up_busfault.c
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+
+#undef CONFIG_DEBUG
+#undef CONFIG_DEBUG_ERROR
+#define CONFIG_DEBUG 1
+#define CONFIG_DEBUG_ERROR 1
```

Comment:
Could you add description why you do those undef and def?

Reply:
like "Enable them to print crash logs always regardless of CONFIG_DEBUG and CONFIG_DEBUG_ERROR"

Reply:
Done.